crates/algokit_transact_ffi/src/lib.rs

```diff
@@ -162,7 +172,7 @@ pub struct Transaction {
     /// The sender of the transaction
     sender: Address,
 
-    fee: u64,
+    fee: Option<u64>,
```

Comment:
Question for the team: Are we ok with making fee Optional. In the case, it's not set it's omitted from transaction when sent.

It's probably worth building a txn with a None fee set, signing, encoding and sending using goal to ensure that it's a valid request.

Reply:
As discussed in today's meeting, optional fee field makes sense and is equivalent to 0 fee